doc/Programming-guide/Server-REST-APIs/swagger.json

```diff
@@ -642,12 +642,13 @@
     "/api/admins/{tenantId}" : {
       "post" : {
         "tags" : [ "User" ],
-        "summary" : "Select tenant admins by tenant id",
-        "description" : "",
+        "summary" : "Get tenant admins based on tenant id",
+        "description" : "Gets the tenant admins by specified tenantId. Only user with KAA_ADMIN role is allowed to perform this operation.",
```

Comment:
This is simple improvement to the doc or smth that is out of scope of this ticket?